.github/workflows/ci.yml

```diff
@@ -81,18 +85,16 @@ jobs:
           !contains(matrix.platform.target, 'android') &&
           !contains(matrix.platform.target, 'ios'))
         run: cargo $CMD test --verbose --target ${{ matrix.platform.target }} $OPTIONS --features $FEATURES
-
-      - name: Build with serde enabled
-        shell: bash
-        run: cargo $CMD build --verbose --target ${{ matrix.platform.target }} $OPTIONS --features serde,$FEATURES
-
-      - name: Build tests with serde enabled
-        shell: bash
-        run: cargo $CMD test --no-run --verbose --target ${{ matrix.platform.target }} $OPTIONS --features serde,$FEATURES
-
-      - name: Run tests with serde enabled
-        shell: bash
-        if: (
-          !contains(matrix.platform.target, 'android') &&
-          !contains(matrix.platform.target, 'ios'))
-        run: cargo $CMD test --verbose --target ${{ matrix.platform.target }} $OPTIONS --features serde,$FEATURES
```

Comment:
why was these removed too?

Reply:
looks like we also don't run tests with `tray` feature enabled, maybe we should add that too.

Reply:
yeah that sounds better

Reply:
I added a new `platform` in matrix: `ubuntu-with-all-features`, which includes all features described in `Cargo.toml`

Reply:
we need to test the tray on all platforms, not just ubuntu, so you should just re-add these 3 steps and add tray to it